### DISCUSSION TITLE
Wiring Question for DS18B20 Sensor with 6-Wire Pump

### DISCUSSION THREAD

**Discussion by phauxcamus** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#657] [Q&A]
Hello! For 6-wire pumps, I saw your answer in #280 about sacrificing audio control (D6) and using that for the DS18B20. There's also mention of this on Page 11 of the manual PDF, but that suggests I can remap the audio control to D0 to retain that function and put the temp sensor on D6. Is that accurate? And do I still need the 4.7K resistor from sensor output to VIN as mentioned in #628?

(I decided to delete this comment from an already-answered question and make it it's own for visibility, I hope that's okay!)

**Comment by visualapproach** [comment]
It's okay.
I haven't tried it myself but you should probably do as it says in the datasheet
